crates/pessimistic-proof-core/src/imported_bridge_exit.rs

```diff
@@ -38,7 +38,7 @@ pub struct L1InfoTreeLeaf {
 
 impl L1InfoTreeLeaf {
     pub fn hash(&self) -> Digest {
-        self.inner.hash()
+        self.inner.hash(keccak256_combine([self.mer, self.rer]))
```

Comment:
but you need the hash leaf to verify it against the l1 info root right?¿ i think this won't work